api_bonus.yaml

```diff
@@ -0,0 +1,375 @@
+          schema:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The identifier of the created request.
+          headers:
+            Location:
+              type: string
+              description: |-
+                Location (URL) of created activation resource.
+                A GET request to this URL returns the request status and details.
+          examples:
+            application/json:
+              id: SOMEBONUSID
+        "409":
+          description: |
+            Cannot activate a new bonus because another bonus related to this user was found.
+        "403":
+          description: |
+            Cannot activate a new bonus because the eligibility data has expired.
+            The user must re-initiate the eligibility procedure to refresh her data
+            and retry to activate the bonus within 24h since her got the result.
+        "401":
+          description: Bearer token null or expired.
+        "500":
+          description: Service unavailable.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+
+    get:
+      operationId: getAllBonusActivations
+      summary: |
+        Get all IDs of the bonus activations requested by
+        the authenticated user or by any between his family member
+      responses:
+        "200":
+          description: |
+            List of bonus activations ID activated or consumed by the authenticated user 
+            or by any between his family members (former and actual)
+          schema:
+            $ref: "#/definitions/PaginatedBonusActivationsCollection"
+        "404":
+          description: No activation found.
+        "401":
+          description: Bearer token null or expired.
+        "500":
+          description: Service unavailable.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+
+  "/bonus/vacanze/activations/{bonus_id}":
+    get:
+      operationId: getLatestBonusActivationById
+      summary: |
+        Get the activation details for the latest version
+        of the bonus entity identified by the provided id
+      parameters:
+        - name: bonus_id
+          in: path
+          required: true
+          description: Bonus activation ID.
+          type: string
+      responses:
+        "202":
+          description: Processing request.
+          schema:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The identifier of the created request.
+          examples:
+            application/json:
+              id: SOMEBONUSID
+        "200":
+          description: Bonus activation details.
+          schema:
+            $ref: "#/definitions/BonusActivation"
+          examples:
+            application/json:
+              id: SOMEBONUSID
+              applicant_fiscal_code: SPNDNL80R11C522K
+              code: MYSECRETCODE
+              qr_code:
```

Comment:
@gunzip 
As we did for the medical prescription, this API should provide svg+png

- svg for display in app
- png to share it as image outside the app

```json
"qr_code": [
    {
      "mime_type": "svg+xml",
      "base64_content": "svgBase64"
    },
    {
      "mime_type": "image/png",
      "base64_content": "pngBase64"
    }
  ]
```